### ISSUE TITLE
Sync with Open Reader Server: inoreader not work.

### ISSUE THREAD

**Issue by neargle** [ViennaRSS/vienna-rss#1201]
I have sync wich www.inoreader.com for one year. But after i update to the new version of vienna, it can not work any more.
Setting of username and password is correct, and i can access https://www.inoreader.com.
How can i debug it?

**Comment by neargle** [comment]
Return v3.4.2 can work here.

**Comment by barijaona** [comment]
Inoreader sync is working in my config.
Can you check in your preferences, quit the application (precaution because of issues #1131, #417 and  #376), then relaunch ?

**Comment by neargle** [comment]
I update today, and sync authentication can not work again. 
![image](https://user-images.githubusercontent.com/7868679/51225323-0c58c600-1985-11e9-91b7-3a7a7c63addb.png)

**Comment by barijaona** [comment]
Could you try zapping Vienna's cookies ?
- Open ~/Library/Cookies in Finder
- Trash the `uk.co.opencommunity.vienna2.binarycookies` file
- Reboot

(You must do it through Finder, because macOS Mojave will prevent you from doing it through command line)

**Comment by neargle** [comment]
> Could you try zapping Vienna's cookies ?
> 
> * Open ~/Library/Cookies in Finder
> * Trash the `uk.co.opencommunity.vienna2.binarycookies` file
> * Reboot
> 
> (You must do it through Finder, because macOS Mojave will prevent you from doing it through command line)

I update to Vienna 3.5.4 and try this, and sync authentication can not work again.

**Comment by barijaona** [comment]
I guess you have some special characters in your password, which require encoding.

Version 3.5.5 will include a fix for this.

**Comment by neargle** [comment]
Yes, you are right.

**Comment by aint** [comment]
I experience the same issue. I was using Vienna with inoreader for years and suddenly sync stopped working today. I didn't change my password and it doesn't contain special characters. 
Tried to downgrade to 3.5.3 and 3.4.2 but it didn't help me.

**Comment by aint** [comment]
I checked out this issue #1203 and here is some logs:
```
default	11:35:58.874640 +0200	Vienna	Task <A32638D9-C85A-460C-BC74-C7F703C7618C>.<1> now using Connection 1
default	11:35:58.874722 +0200	Vienna	Task <A32638D9-C85A-460C-BC74-C7F703C7618C>.<1> sent request, body S
default	11:35:59.339110 +0200	Vienna	Task <A32638D9-C85A-460C-BC74-C7F703C7618C>.<1> received response, status 403 content K
default	11:35:59.339414 +0200	Vienna	Task <A32638D9-C85A-460C-BC74-C7F703C7618C>.<1> response ended
default	11:35:59.339753 +0200	Vienna	clientAuthOperation statusCode 403 for URL https://www.inoreader.com/accounts/ClientLogin?accountType=GOOGLE&service=reader
default	11:35:59.339962 +0200	Vienna	Headers {
    "Content-Encoding" = gzip;
    "Content-Length" = 87;
    "Content-Type" = "text/html; charset=UTF-8";
    Date = "Mon, 11 Mar 2019 09:35:59 GMT";
    Server = "nginx/1.14.2";
    Vary = "Accept-Encoding,User-Agent";
    "x-powered-by" = "PHP/5.6.40";
}

```

So I get 403 but my password is defenitely correct.  Can I somehow debug what exectly Vienna sends to inoreader?

**Comment by barijaona** [comment]
As explained above, the problem is related to some special characters like /:?#[]@!$&'()*+,;=

A fix has already been included in the current master code.
If you have Xcode 10 installed on a machine, you can build Vienna and check if the problem is solved.

Otherwise, a temporary workaround would be to change your password to avoid problematic characters.
But hopefully, version 3.5.5 will be available very soon !

**Comment by aint** [comment]
@barijaona thanks for the clarification. But the problem is that my password contains only digits. Also I've builded the app from master and issue still persists.  


<img width="688" alt="image" src="https://user-images.githubusercontent.com/3179252/54151028-8caa1d80-4442-11e9-9ef8-99624a5e0243.png">

**Comment by cloudysulphur** [comment]
i have the same issue as aint.  The issue also started yesteday.  There are no special characters in the password (random password generated by password vault).

I don't have a paid inoreader subscription,  and they are changing plans lately.  I am not sure if the problem is related to vienna or inoreader.

**Comment by tonycpsu** [comment]
For those who had troubles unrelated to special characters in their passord, see #1252.

**Comment by barijaona** [comment]
The password problem has been closed by version 3.5.5
